katalog/curator-s3/curator.yml

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2020 SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: curator
+spec:
+  schedule: "15 0 * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: curator-add-task
+          initContainers:
+            - name: curator-add-task
+              image: quay.io/sighup/kubectl:v1.13.2
```

Comment:
This container image definition has to be moved to the kustomization.yaml.
Also, it could be a good idea to update the image (from v1.13 -> v1.17)

Reply:
i don't get this point, can you explain it better?

Reply:
All the images have to be defined in the kustomization.yaml even if it is only a init container.
Take this as an example:

```yaml
namespace: logging

resources:
- curator.yml

images:
- name: quay.io/sighup/curator
  newName: reg.sighup.io/sighupio/fury/curator
  newTag: 5.8.3_3.8-alpine_3.13

configMapGenerator:
- name: curator
  files:
  - action.yml
  - config.yml
 ```

Reply:
done